tests/unit/test_spaces.cpp

```diff
@@ -282,6 +282,7 @@ TEST_P(FP32SpacesOptimizationTest, FP32L2SqrTest) {
 
     dist_func_t<float> arch_opt_func;
     float baseline = FP32_L2Sqr(v, v2, dim);
+#ifdef OPT_AVX512F
     if (optimization.avx512f) {
```

Comment:
Nice! We should have done it in this order on the spaces files as well :(

Reply:
Yeah i wondered if there's a good reason for that
I'll change it 🤫